test-scaling/script.sh

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env bash
+
+registry='francisregistry.azurecr.io'
+image_tag='francis-test-scaling'
+container_repo="${registry}/gadget"
+full_image_name="${registry}/stress_exec:latest"
+
+function prepare {
+	docker build -t $full_image_name -f Dockerfile .
+	docker push $full_image_name
+
+	CONTAINER_REPO=$container_repo make -C .. gadget-default-container
+	CONTAINER_REPO=$container_repo make -C .. push-gadget-container
+}
+
+function prepare_csv_file {
+	local csv_file=$1
+	local nodes_nr=$2
+
+	echo -n "" > $csv_file
+
+	# Create CSV header
+	for i in $(seq 1 ${nodes_nr}); do
+		echo -n "node-${i}," >> $csv_file
+	done
+	echo "gadget" >> $csv_file
+}
+
+function pre_run {
+	local namespace=$1
+
+	kubectl create ns $namespace
+	../kubectl-gadget deploy --image="${container_repo}:${image_tag}"
+}
+
+function run {
+	local nodes_nr=$1
+	local iter=$2
+	local namespace=$3
+	local gadget_dir=$4
+	local csv_file=$5
+
+	gadget_file="${gadget_dir}/gadget-output-${nodes_nr}-${iter}.out"
+	../kubectl-gadget trace exec -n $namespace -o json > $gadget_file &
+	gadget_pid=$!
+
+	kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: daemonset-${nodes_nr}
+  namespace: $namespace
+spec:
+  selector:
+    matchLabels:
+      k8s-app: stress-exec
+  template:
+    metadata:
+      labels:
+        k8s-app: stress-exec
+    spec:
+      initContainers:
+      - name: stress-exec
+        image: $full_image_name
```

Comment:
Any reason not  to use `ghcr.io/colinianking/stress-ng:master` directly here and pass the command?